docs/static-pvc.md

```diff
@@ -0,0 +1,117 @@
+# Static PVC with ceph-csi
+
+- [Static PVC with ceph-csi](#static-pvc-with-ceph-csi)
+  - [RBD static PVC](#rbd-static-pvc)
+    - [Create RBD image](#create-rbd-image)
+    - [Create RBD static PV](#create-rbd-static-pv)
+    - [RBD Volume Attributes in PV](#rbd-volume-attributes-in-pv)
+    - [Create RBD static PVC](#create-rbd-static-pvc)
+
+This document outlines how to create static PV and static PVC from existing rbd image
+
+**warning** static PVC can be created, deleted, mounted and unmounted but
+doesn't support other operations like (snapshot, resize, etc)
+
+## RBD static PVC
+
+RBD images created manually can be mounted and unmounted to an app, below step
+shows how to create a rbd image, static PV, static PVC
+
+### Create RBD image
+
+Lets create a new rbd image in ceph cluster which we are going to use for
+static PVC
+
+```bash
+[$]rbd create static-image --size=1024 --pool=replicapool
+```
+
+### Create RBD static PV
+
+To create the rbd PV you need to know the `rbd image name`,`clusterID` and
+`pool` name in which the rbd image is created
+
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: fs-static-pv
+spec:
+  accessModes:
+  - ReadWriteOnce
+  capacity:
+    storage: 1Gi
+  csi:
+    driver: rbd.csi.ceph.com
+    fsType: ext4
+    nodeStageSecretRef:
+      # node stage secret name
+      name: csi-rbd-secret
+      # node stage secret namespace where above secret is created
+      namespace: default
+    volumeAttributes:
+      # Required options from storageclass parameters need to be added in volumeAttributes
+      clusterID: ba68226a-672f-4ba5-97bc-22840318b2ec
+      imageFeatures: layering
```

Comment:
Just curious, why does one need to specify image features for an image that is to be created and removed manually?

Reply:
This is not mandatory, i have added a table below about required flags, anyhow will remove it